rental.rb

```diff
@@ -0,0 +1,13 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date)
```

Comment:
- [ ] The constructor of your Rental should be modified so Book and Person are set in it to break the many-to-many relationship: `def initialize(date, book, person)`